Meadowlark-js/packages/meadowlark-utilities/src/Config.ts

```diff
@@ -53,7 +53,8 @@ export type ConfigKeys =
   | 'FASTIFY_RATE_LIMIT'
   | 'HTTP_PROTOCOL_AND_SERVER'
   | 'DISABLE_LOG_ANONYMIZATION'
-  | 'MONGODB_MAX_NUMBER_OF_RETRIES';
+  | 'MONGODB_MAX_NUMBER_OF_RETRIES'
+  | 'AWS_REGION';
```

Comment:
In here by accident?

Reply:
Whoops, yeah I had a bunch of code from my other work accidentally get included so I had to strip it out, must've forgotten one.